src/languageservice/services/yamlCompletion.ts

```diff
@@ -63,6 +64,20 @@ interface CompletionsCollector {
   getNumberOfProposals(): number;
   result: CompletionList;
   proposed: { [key: string]: CompletionItem };
+  context: {
```

Comment:
The collector is sent to each internal function, so let's add some context data so we can distinguish between some specific yaml situation